CODEOWNERS

```diff
@@ -116,6 +116,7 @@
 /boards/shields/atmel_rf2xx/              @nandojve
 /boards/shields/esp_8266/                 @nandojve
 /boards/shields/inventek_eswifi/          @nandojve
+/boards/shields/nrf7002ek/                @krish2718
```

Comment:
We don't use the codeowners file AFAIK, this should be added to the MAINTAINERS.yml file :)

Reply:
Ah I was just following other shields, there is already a top level entry for `shields` in `MAINTAINERS.yml` with except for `Drivers: CAN` covering the shields too.

Reply:
I can follower the `Drivers: CAN` model, so, dropped that commit, once we introduced the driver, I can add this shield to that entry in `MAINTAINERS.yml`.